sdk/storage/azure-storage-datamovement/samples/datamovement_getting_started.cpp

```diff
@@ -15,7 +15,7 @@ std::string GetConnectionString()
   {
     return ConnectionString;
   }
-  const static std::string envConnectionString = std::getenv("AZURE_STORAGE_CONNECTION_STRING");
+  const static std::string envConnectionString = "DefaultEndpointsProtocol=https;AccountName=emmahktest;AccountKey=9J/G/orROAbao1ujMkn4+5l+ZosLXtcY9YXnuX+urcw5tqtkPBm/jf+7w7rb5zvBVdxo1T8DW/vkf6o4KgZgiw==;EndpointSuffix=core.windows.net";
```

Comment:
This will most likely get us flagged by security, if not yet. Then there is a long process ensuring the resource is deactivated, and maybe even git history cleanup.